variables.tf

```diff
@@ -157,7 +157,7 @@ variable "operator_namespace" {
 }
 
 variable "install_metrics_server" {
-  description = "Whether to install the metrics-server for the Materialize Console"
+  description = "Whether to install the metrics-server for the Materialize Console. Defaults to false since GKE installs one by default in the kube-system namespace."
```

Comment:
We could also remove this as a variable ... since if they set to true, it'll error. but eh.

Reply:
If it's invalid to set this to true we should just remove it.